subsys/bluetooth/audio/vocs.c

```diff
@@ -53,7 +53,7 @@ static ssize_t write_location(struct bt_conn *conn, const struct bt_gatt_attr *a
 			      const void *buf, uint16_t len, uint16_t offset, uint8_t flags)
 {
 	struct bt_vocs *inst = BT_AUDIO_CHRC_USER_DATA(attr);
-	uint32_t old_location = inst->srv.location;
+	uint32_t new_location;
```

Comment:
We could consider using the `enum bt_audio_location` in VOCS, and use that here. That way it also makes more sense to use `BT_AUDIO_LOCATION_RIGHT_SURROUND` instead of `0x08000000`